sdk/tests/deqp/modules/shared/glsFboCompletenessTests.js

```diff
@@ -737,7 +737,7 @@ goog.scope(function() {
                     testFailedOptions('Framebuffer checked as incomplete, but with wrong status', true);
                 }
             } else if (fboStatus != gl.FRAMEBUFFER_COMPLETE && statuses.isFBOStatusValid(gl.FRAMEBUFFER_COMPLETE)) {
-                testFailedOptions('Framebuffer object could have checked as complete but did not.', true);
+                testPassedOptions('Warning: framebuffer object could have checked as complete but did not.', true);
```

Comment:
I believe the deqp tests actually overdo by issuing a warning.  Here is the situation.  The test is for framebuffer with multiple attachments, each with different "samples". When the framebuffer is tested to be incomplete, the tests issue a warning, saying that the fbo could be complete if the underlying driver actually upgrade all the "samples" to the same "RENDERBUFFER_SAMPLES" (which is allowed by the spec). This is very unlikely and I think WebGL2 spec should just return INCOMPLETE to avoid different behaviors among driver implementations and the tests should just verify INCOMPLETE is the status, but that would need to be agreed by other WebGL2 implementors.